composer.json

```diff
@@ -5,15 +5,15 @@
     "license": "MIT",
     "require": {
         "php": "^7.4 || ^8.0",
-        "sonata-project/admin-bundle": "^4",
-        "symfony/config": "^4.4",
-        "symfony/console": "^4.4",
-        "symfony/security-core": "^4.4",
+        "sonata-project/admin-bundle": "^4.21.0",
```

Comment:
Sonata Admin v4.21.0 is gereleased en bevat mijn bugfix. Dus de tijdelijke fix met een `__clone()` method in `src/Admin/ContentItemAdmin.php` kan er nu uit

Reply:
Fix verplaatst naar #100